index.html

```diff
@@ -67,6 +67,12 @@ <h1 class="title">CE EB Utils</h1>
           <i class="fa fa-angle-right fa-lg"></i>
         </a>
       </li>
+      <li class="pure-menu-item">
+        <a id="sqlValidatorMenu" href="html/sqlvalidator/sqlvalidator.html" class="pure-menu-link">
+          SQL Validator
```

Comment:
Would prefer this to be more generic `Validator` so we can add others easily (swagger, element, etc)